packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -755,9 +755,9 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
         }
         this.updateButton(button, options);
 
-        setTimeout(() => {
+        requestAnimationFrame(() => {
```

Comment:
Side-note: We need to handle these cases more elegantly, but agree `requestAnimationFrame()` is probably closer to the target than `setTimeout()`.

Reply:
We need to ensure the styles were applied, and setTimeout might be called before that